docs/rulesets.md

```diff
@@ -1,41 +1,128 @@
----
-title: Spectral Rulesets
-tags: 
-- Documentation
-- Rulesets
----
+# Spectral Rulesets
 
-Rulesets are a container for collections of rules. These rules are essentially calling functions Spectral, and by taking parameters you can make these functions do whatever you want. 
+Rulesets are a container for collections of rules. These rules are taking parameters, and calling functions on certain parts of the JSON object being linted.
 
-First the rule filters your object (JSON/YAML file) down to a set of target values, and then list what function arguments should be passed in.
+## CLI Usage
```

Comment:
Maybe link to the CLI docs here too

Reply:
Do you mean on the title itself?
```suggestion
## [CLI Usage](./cli.md)
```

Reply:
Usually I avoid links in headers, I should have specified "here" as in that section somewhere.

Reply:
Makes sense. So let's put a sentence such as 

> To get an overview of the CLI chek out the [documentation page](link)

?

Reply:
👍 

(minus "chek" ;) )